### PR TITLE
 Adding check for insufficient number of nodes

### PR DESCRIPTION
**Pull request by prernasony**
Adding check for number of nodes less than 3

### CODE REVIEW

**Comment by ramkrsna** [comment]
Thank you for the PR. We need to make it into a function to check the valid number of nodes. that way we can call it each condition. The config file is going to be made into a jinja template. either way we can handle if its a function.

File: cic.py [hunk 1]
```diff
@@ -108,7 +108,11 @@ def both_in_use():
         print (60 * '-')
         avail_hosts = int(raw_input("How many nodes are available ?:  "))
 
-        if avail_hosts >= 4:
+	if avail_hosts < 3 :
+		print( " Invalid number of nodes. Try again... " )
```
Comment (sankarshanmukhopadhyay) [inline cic.py]: Suggest that we use messages "less the recommended number of nodes" or, similar.

File: cic.py [hunk 2]
```diff
@@ -77,6 +77,12 @@ def both_in_use():
             print "\033[91m %s \033[0m is not valid hostname" % hosts
             exit()
 
+def is_valid_node_count():
+    if avail_hosts < 3 :
+            print( " Less than recommended/minimum number of nodes. Try again... " )
```
Comment (mykaul) [inline cic.py]: Should be: 'Less than the minimum (3) number of nodes'.

File: cic.py [hunk 3]
```diff
@@ -108,7 +114,8 @@ def both_in_use():
         print (60 * '-')
         avail_hosts = int(raw_input("How many nodes are available ?:  "))
 
-        if avail_hosts >= 4:
+	is_valid_node_count()
+	if avail_hosts >= 4:
```
Comment (mykaul) [inline cic.py]: Why do we need this check now? We are already checking we have enough hosts?

File: cic.py [hunk 4]
```diff
@@ -77,6 +77,12 @@ def both_in_use():
             print "\033[91m %s \033[0m is not valid hostname" % hosts
             exit()
 
+def is_valid_node_count():
+    if avail_hosts < 3 :
```
Comment (mykaul) [inline cic.py]: Isn't the minimum 4?

File: cic.py (hunk 2 above)
Comment (prernasony) [reply cic.py]: Okay, I will make this change

**Comment by ramkrsna** [comment]
https://github.com/red-hat-storage/openshift-cic/commit/7ffccbd246a10669de7c98944285e8fb070a9886 commit ( #PR52 ) has addressed this issue. Hence closing the PR

**Pull request closed without merging**